src/_N001_Email.ino

```diff
@@ -93,6 +93,8 @@ boolean NPlugin_001_send(const NotificationSettingsStruct& notificationsettings,
     mailheader.replace(String(F("$ato")), notificationsettings.Receiver);
     mailheader.replace(String(F("$subject")), aSub);
     mailheader.replace(String(F("$espeasyversion")), String(BUILD));
+    String bMesg = aMesg;
```

Comment:
Please do not copy it here.
Remove the 'const' from the function parameter and use aMesg.
It is already a constructed string not used after this sending.
The message body can be quite long and copying large string gives a strain on free memory.

Reply:
Good catch, updating.